rakelib/default_plugins.rb

```diff
@@ -152,7 +152,7 @@ def self.fetch_all_plugins
 
     def self.is_released?(plugin)
       require 'gems'
-      !Gems.search(plugin).empty?
+      !Gems.search(plugin).reject{ |h| h['name'] != plugin }.empty?
```

Comment:
The rubygems gem seems to have an exact search request called [info](https://github.com/rubygems/gems/blob/master/lib/gems/client.rb#L25-L28) so maybe we could use that instead of doing a query and double checking afterwards